examples/docs/es/date-picker.md

```diff
@@ -92,14 +92,15 @@
 </style>
 
 ## DatePicker
+Utiliza Date Picker para introducir fecha.
 
-Use Date Picker for date input.
+###  Entrar fecha
 
-###  Enter Date
+Básica fecha escoja moderado por 'día'.
```

Comment:
For example, this is confusing, but we can correct it later

Reply:
Maybe Fecha escoja básica moderado por 'day'. better?

Reply:
I think this is better: `Selector de fecha básico limitado por día`

Reply:
OK,thanks,I will modify the datetime-picker as well.